### PR TITLE
Changes on the specify origin option

### PR DESCRIPTION
**Pull request by stevenndungu**
I have changed the format of origin from "30-12-1899" to "1899-12-30" for both excel and Gregorian... after noticing an anomaly in the output given.
i.e

as.Date(42034,origin="30-12-1899 (Excel)")
[1] "0146-01-18"
> as.Date(42034,origin="1899-12-30(Excel)")
[1] "2015-01-30"

 
And too put a constrain on the Single column (when specify origin is checked) to only accept numeric variables to avoid error message in case a different datatype is provided for conversion.

### CODE REVIEW

**Comment by dannyparsons** [comment]
@stevenndungu Yes, you are right that is giving the correct dates.
However, we would still like the combo box to display "30-12-1899 (Excel)" because that's helpful to the user, but it should pass in `origin = "1899-12-30"` to `as.Date` because that's the correct format for R.
(Although `origin="1899-12-30(Excel)` works, it's really the way origin is meant to be used)
@Lunalo has done this in other dialog so can help with this if you need it.

About having the receiver only as numeric when origin is specified, that's a great idea to avoid the error.

**Comment by stevenndungu** [comment]
Nice, will check on that with Lunalo tomorrow. Thanks.

File: instat/dlgMakeDate.vb [hunk 1]
```diff
@@ -81,6 +79,14 @@ Public Class dlgMakeDate
             ucrBase.OKEnabled(False)
         End If
     End Sub
+    Private Sub DateFormat()
+        Select Case ucrInputOrigin.GetText
+            Case "30-12-1899 (Excel)"
+                ucrBase.clsRsyntax.AddParameter("origin", Chr(34) & "1899-12-30(Excel)" & Chr(34))
+            Case Else
+                ucrBase.clsRsyntax.AddParameter("origin", Chr(34) & "1600-01-03 (Gregorian)" & Chr(34))
```
Comment (dannyparsons) [inline instat/dlgMakeDate.vb]: I would like the parameter that's passed to only have the date string not the extra (Excel) or (Gregorian) part at the end.

**Comment by stevenndungu** [comment]
Ok can delete that other part

**Comment by stevenndungu** [comment]
You are right. It shouldn't be there actually. Was a typo error.

**Comment by dannyparsons** [comment]
Is this ready now?

**Comment by stevenndungu** [comment]
Yes its ready, I have made corrections suggested and tested it. Its working
fyn.

**Comment by stevenndungu** [comment]
Great

**Pull request merged**